common/endpoints/endpoints.go

```diff
@@ -6,38 +6,55 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"time"
 
 	"github.com/scootdev/scoot/common/stats"
 )
 
-func RegisterHealthCheck(path string) {
-	http.HandleFunc(path, healthHandler)
+func NewTwitterServer(addr string, stats stats.StatsReceiver) *TwitterServer {
```

Comment:
nit: I'd prefer to not tie this to Twitter and would instead use ReportsServer or similar. Not a big deal though.